configuration/config-txt/video.md

```diff
@@ -642,8 +659,9 @@ The options that can be set are:
 
 ### test_mode
 
-The `test_mode` command displays a test image and sound during boot (over the composite video and analogue audio outputs only) for the given number of seconds, before continuing to boot the OS as normal. This is used as a manufacturing test; the default value is `0`.
+The `test_mode` command outputs a test image on the composite video output, and sound during boot from the analogue output, for the given number of seconds, before continuing to boot as normal. This is used as a manufacturing test; the default value is `0`.
```

Comment:
As it already says "before continuing to boot as normal", perhaps the first "during boot" is redundant?

Reply:
Fixed.